server/app/dal/schemas/category.py

```
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+
+class Category(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)